build.assets/robotest/pr_config.sh

```diff
@@ -55,7 +55,7 @@ EOF
 
 function build_install_suite {
   local suite=''
-  local test_os="redhat:7"
+  local test_os="centos:7"
```

Comment:
I wonder if we can replace one of the numerous upgrade scenarios here with an additional install on RedHat (so keep RHEL in addition to CentOS)? I think testing RHEL installs is important - a lot of customers are running on it. For example, we can throw away upgrade from 6.1.0 - I think that realistically nobody's using it anymore so it's wasted cycles.

Reply:
We still have RHEL on shrink.  Do you want a vanilla install in addition to that?

I know I only showed the diff above.  Here is the full pr config:

```
walt@work:~/git/gravity$ ./build.assets/robotest/pr_config.sh
install={"flavor":"three","nodes":3,"role":"node","os":"centos:7","storage_driver":"overlay2"}
install={"installer_url":"/installer/opscenter.tar","nodes":1,"flavor":"standalone","role":"node","os":"ubuntu:18","ops_advertise_addr":"example.com:443"}
resize={"to":3,"flavor":"one","nodes":1,"role":"node","state_dir":"/var/lib/telekube","os":"ubuntu:18","storage_driver":"overlay2"}
shrink={"nodes":3,"flavor":"three","role":"node","os":"redhat:7"}
upgrade={"flavor":"three","nodes":3,"role":"node","os":"debian:9","from":"/telekube_6.1.0.tar","service_uid":997,"service_gid":994,"storage_driver":"overlay2"}
upgrade={"flavor":"three","nodes":3,"role":"node","os":"ubuntu:16","from":"/telekube_7.0.7.tar","service_uid":997,"service_gid":994,"storage_driver":"overlay2"}
upgrade={"flavor":"three","nodes":3,"role":"node","os":"centos:7","from":"/telekube_6.3.18.tar","service_uid":997,"service_gid":994,"storage_driver":"overlay2"}
upgrade={"flavor":"three","nodes":3,"role":"node","os":"centos:7","from":"/telekube_7.0.13.tar","service_uid":997,"service_gid":994,"storage_driver":"overlay2"}
upgrade={"flavor":"three","nodes":3,"role":"node","os":"ubuntu:18","from":"/telekube_7.0.12.tar","service_uid":997,"service_gid":994,"storage_driver":"overlay2"}
upgrade={"flavor":"three","nodes":3,"role":"node","os":"redhat:7","from":"/telekube_6.1.32.tar","service_uid":997,"service_gid":994,"storage_driver":"overlay2"}
```

Reply:
As for 6.1.0, you're right that we don't care about that specific version.  6.1.0 is more of a placeholder for "all 6.1 versions other than the most recent", and generally the older we go the more differences we exercise.

I'd be hesitant to say "we only need to test 6.1.latest to 7.0", as most users aren't running 6.1.latest either, they're somewhere in between.

Reply:
I guess it should be fine then, since shrink includes a 3-node install as well.